addons/project/wizard/project_report_wizard.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class ProjectReportWizard(models.TransientModel):
+    _name = 'project.report.wizard'
+    _description = 'Project Report Wizard'
+
+    project_id = fields.Many2one('project.project', required=True)
+    report_type = fields.Selection([
+        ('burndown', 'Burndown Chart'), ('project_update', 'Project Update')
+    ], string='Report Type', required=True, default='burndown')
+
+    def action_burndown_report_view(self):
+        action = self.env['ir.actions.act_window']._for_xml_id('project.action_project_task_burndown_chart_report')
+        action['context'] = {'search_default_project_id': self.project_id.id}
```

Comment:
In order not to overwrite an eventual context in the action, I would rather do this otherwise you will lose your `default_report_type` context key:
```suggestion
        action['context'] = dict(literal_eval(action.get('context'), {}), search_default_project_id=self.project_id.id)
```

Reply:
Ooops I made a mistake in the diff, sorry. Isn't it the reason of the error?
In order not to overwrite an eventual context in the action, I would rather do this otherwise you will lose your `default_report_type` context key:
```suggestion
        action['context'] = dict(literal_eval(action.get('context', {})), search_default_project_id=self.project_id.id)
```

Reply:
@IT-Ideas , It will work for this action(project update action) as there is no `active_id ` in the context, it's blank dict. but for [burndown chart action](https://github.com/odoo/odoo/pull/75509#discussion_r759189801), it will not work as we have `active_id` in the context.

Thanks

Reply:
Ok I see Thanks 👍🏼 
I think that this is indeed getting a bit overkill.
Let's overwrite the context in the two actions then 🙂